starter/script.js

```diff
@@ -1 +1,68 @@
 // put code here!
+
+var welcome = prompt("Congratulations!  You've been beamed into a Hardy Boys mystery.  Choose a character: \n('f') Frank \n('j') Joe");
+//Opening prompt.  Note: choosing Frank or Joe will not make a diffrerence.
+console.log("Class at Bayport High has just let out and you receive an urgent message from your dad.  A rare moonstone  shipped on a freighter just arriving from India has disappeared.");
+var locations = prompt("Where do you want to go? \n('h') Home- Talk things over with your private detective father \n('p') Police Station- get the low-down from Chief Collig \n('x') Wharf- Head down to the wharf and check it out");
+//You can choose between 3 locations
+if (locations === 'h') {
+  console.log("You catch sight of a suspicious character eavesdropping in the bushes.  What do you do?");
+  var decisionHome = prompt("What do you do? \n('c') Chase him away! \n('i') Ignore him, act like everything is normal and plant false information in your meeting with Dad");
+  if (decisionHome === 'c') {
+    console.log("The man sees you and gets away!  You return home, where your dad gives you the details of the theft.  It is a complex smuggling operation, and because all the trails are cold, there is no clear next step.  Aw, shucks!");
+    }
+  else if (decisionHome === 'i') {
+    console.log(" Great thinking, sleuth!  Dad outlines a false plan to arrange a meeting with the ship's crew.  The police close in on the warehouse and intercept the theif on his way in.  The moonstone is recovered!");
+    var celebrate = prompt("How do you want to celebrate? \n('p') Beach Boys dance party at Callie's house. \n('d') Call Chet, Tony, Biff, Callie, and Iola and meet up at the Silver Diner.");
+    if (celebrate === 'p') {
+      console.log("Woohoo!  Mystery solved.");
+    }
+    else if (celebrate === 'd') {
+      console.log("Burgers and shakes!  Mystery solved.");
+    }
+  }
```

Comment:
How would you go about dealing with user inputs that don't match to any of these specific cases?